tensorrt_llm/_torch/pyexecutor/llm_request.py

```diff
@@ -566,6 +568,8 @@ def __init__(
         # currently, keep py_stop_words_list as python list, rather than tensor.
         self.py_stop_words_list = stop_words_list
 
+        self.py_logprobs_mode = LogprobMode.RAW if logprobs_mode is None else logprobs_mode
```

Comment:
If so, can we make the default as `RAW` instead of None in the function declaration?